docs/keepassxc-browser-manual.md

```diff
@@ -0,0 +1,333 @@
+        - [1.3. Detection of credential fields](#13-detection-of-credential-fields)
+        - [1.4. Choose custom credential fields for a page](#14-choose-custom-credential-fields-for-a-page)
+        - [1.5. Create new or modify existing credentials](#15-create-new-or-modify-existing-credentials)
+        - [1.6. Choose credentials for HTTP Basic Auth requests](#16-choose-credentials-for-http-basic-auth-requests)
+        - [1.7. Fill additional values using String Fields](#17-fill-additional-values-using-string-fields)
+        - [1.8. Fill TOTP](#18-fill-totp)
+- [2. Configuration and settings](#2-configuration-and-settings)
+    - [2.1. Settings: General](#21-general)
+    - [2.2. Settings: Connected databases](#22-connected-databases)
+    - [2.3. Settings: Custom credential fields](#23-settings-custom-credential-fields)
+    - [2.4. Settings: Site preferences](#24-settings-site-preferences)
+- [3. KeePassXC browser integration settings](#3-keepassxc-browser-integration-settings)
+    - [3.1. General](#31-general)
+    - [3.2. Advanced](#32-advanced)
+- [4. Troubleshooting](#4-troubleshooting)
+
+
+### 1. Functionality
+
+This section explains the basic functionality of KeePassXC-Browser.
+
+#### 1.1. Access the credentials
+If KeePassXC-Browser detects an input field on a web page, a request is sent to KeePassXC to check if the page URL matches with any entry in the database.
+The received credentials are accesible via multiple ways which are described in the following sections.
+
+##### 1.1.1. Popup
+The extension icon background is green and it is shown with a question mark. Clicking on the icon opens a popup on where you can choose the credentials to use.
+
+<div style="text-align: center;">
+<img alt="KeePassXC-Browser credentials popup" style="max-width: 100%;"
+src="{{ site.baseurl }}/images/browser-migration/manual_credentials_popup.png">
+</div>
+
+##### 1.1.2. Autocomplete
+Username field supports autocomplete for showing a list of received credentials. If there is only a password field visible on the page, autocomplete will be still available.
+By clicking the input field, the autocomplete menu is visible and correct username can be selected. Password field will be filled automatically.
+This feature is activated by default and can be disabled on the *General* settings page.
+
+<div style="text-align: center;">
+<img alt="KeePassXC-Browser credentials autocomplete" style="max-width: 100%;"
+src="{{ site.baseurl }}/images/browser-migration/manual_credentials_autocomplete.png">
+</div>
+
+##### 1.1.3. Context menu
+Context menu is accesible on any input field, even when no credentials are detected for the page.
+The following options are available:
+
+**Fill User & Pass** will only work if KeePassXC-Browser received only one pair of credentials. Otherwise it shows you a notification and autocomplete or popup should be used instead.
+**Fill Pass only** will work similary, but fills only the password of a single pair of credentials.
```

Comment:
"similary" → "similarly"
For this and the above line, I'd make the tense into "**Fill User & Pass** only works if…" and "**Fill Pass only** works similarly…", like the "**Fill TOTP**" entry (i.e. "works" instead of "will work").